module_modo.lua

```diff
@@ -249,6 +249,10 @@ function Module:HandleEmojiAdd(userId, message)
 			content = content:sub(1, 800) .. "...<truncated>"
 		end
 
+		if (content ~= nil) then
```

Comment:
The check is wrong, this is checking if there's content and overriding it if so.

Reply:
Plus, this check is not required as it's already done on line 271
```lua
					value = content or "<empty>"
```
I suppose Discord rejects valid but empty messages (which could be checked with `(content and #content > 0) and content or "empty"`)